### PR TITLE
[SRVKE-1168] Kafka Broker monitoring dashboard

### PR DESCRIPTION
**Pull request by pierDipi**
As per title, see individual commits for details

Some screenshots:

broker-dashboard-1
broker-dashboard-2
broker-dashboard-3
broker-dashboard-4
broker-dashboard-5

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml [hunk 1]
```diff
@@ -1364,7 +1364,7 @@ data:
         ]
       },
       "timezone": "",
-      "title": "Knative Eventing - Broker/Trigger",
+      "title": "Knative Eventing - Channel-based Broker/Trigger",
```
Comment (skonto) [inline knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml:1367]: This requires changes in the docs, for example https://docs.openshift.com/container-platform/4.9/serverless/admin_guide/serverless-admin-monitoring.html
```
Select the Knative Eventing - Broker/Trigger dashboard in the Dashboard drop-down list.
```
Let's open a jira for this too.

**Comment by skonto** [comment]
@pierDipi lgtm have you tested it with multiple sources in different namespaces?

**Comment by pierDipi** [comment]
> @pierDipi lgtm have you tested it with multiple sources in different namespaces?

Yes, I use this https://github.com/knative-sandbox/eventing-kafka-broker/pull/1417/files + the existing https://github.com/knative-sandbox/eventing-kafka-broker/tree/main/test/config/sacura for generating load.

as you can see from my screenshots, there is a line at 100 events/s and then after a while it goes to 200 events/s (that is when the other job in the other namespace starts)

File: knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml (hunk 1 above)
Comment (pierDipi) [reply knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml:1367]: Let me capture a Jira and I will work on it next (if the changed title makes sense to you).

Comment (pierDipi) [reply knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml:1367]: https://issues.redhat.com/browse/SRVKE-1173

**Pull request merged**